packages/medusa/src/loaders/__tests__/models.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { createMedusaContainer } from "medusa-core-utils"
+import path from "path"
+
+import modelsLoader from "../models"
+
+describe("models loader", () => {
+  const container = createMedusaContainer()
+  let models
+  let error
+
+  beforeAll(async () => {
+    try {
+      models = await modelsLoader({
+        container,
+        isTest: true,
+        coreTestPathGlob: "../models/{product,product-variant}.ts",
+        rootDirectory: path.join(__dirname, 'customizations'),
+        extensionPathGlob: "models/{product,product-variant}.ts",
+      })
+    } catch (e) {
+      error = e
+    }
+  })
+
+  it("error should be falsy", () => {
+    expect(error).toBeFalsy()
+  })
+
+  it("ensure that the product model is an extended model", () => {
+    expect(models).toHaveLength(2)
+    const productModel = models.find(model => model.name === 'Product')
+
+    expect(new productModel().custom_attribute).toEqual("test")
+  })
+
```

Comment:
Shouldn't we have the isExtendedModel as well since this is what we check on the next test for the other model? It act as the truth prop